tests/unit_tests/rng/include/statistics_check_test.hpp

```diff
@@ -44,6 +44,7 @@
 
 #define POISSON_ARGS 0.5
 
+using namespace cl;
```

Comment:
Adding @aelizaro

Reply:
@aelizaro looks like all real changes are in RNG domain. Could you please review?

Reply:
Thanks @mmeterel, looks good to me!